Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -634,6 +635,77 @@ sourceToDestinationConfig state = let
     }
   in sourceToDestinationConfig'
 
+headerTextView :: forall w.(Action -> Effect Unit) -> DriverInfoCardState -> PrestoDOM (Effect Unit) w 
+headerTextView push state = 
+  linearLayout
+  [ orientation VERTICAL
+  , height WRAP_CONTENT
+  , width MATCH_PARENT
+  , gravity CENTER_VERTICAL
+  , padding $ Padding 16 20 16 16
+  , visibility if (state.props.currentStage == RideAccepted || state.props.currentStage == RideStarted) then VISIBLE else VISIBLE
```

Comment:
@saireddy2610 What is the use of this condition both if and else condition is `VISIBLE`

Reply:
i will remove the line